### DISCUSSION TITLE
MoneyExtension

### DISCUSSION THREAD

**Discussion by sebastiaosiqueira** [balta-io/forum#710] [Tira Dúvidas] [answered]
Fundamentos do Azure, Git, GitHub e DevOps
Fiquei com uma dúvida na hora de criar o arquivo yml. conforme abaixo a versao coloquei 6.0.0 como instrução

      steps:
        - uses: actions/checout@main

        - name: Set up .Net Core
          uses: actions/setup-dotnet@v2
          with:
            **dotnet-version: '6.0.x'**

mas eu entrando dentro da pasta MoneyExtension  bin existe uma pasta net8.0. 
![moneyextension](https://github.com/user-attachments/assets/6c7021fa-5e99-4865-a4d9-31e3b86b3da0)

Qual versão coloco? ou pode subir assim?

**Comment by andrebaltieri** [comment, marked as answer]
@sebastiaosiqueira isto ocorre pois localmente você deve estar usando o .NET 8, mas no seu YAML definiu versão 6.
Você pode alterar a versão para 8 no YAML ou usar a versão 6 local!
